src/App.jsx

```diff
@@ -1,107 +1,29 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList/PostList';
+
+function getUserById(userId) {
+  return usersFromServer.find(user => user.id === userId)
+      || null;
+}
+
+function getCommentById(postId) {
+  return commentsFromServer.filter(comment => comment.postId === postId)
+      || null;
```

Comment:
No need in `|| null` because `filter` always returns an array, which is always a truthy value